nilearn/input_data/masker_validation.py

```diff
@@ -0,0 +1,75 @@
+import warnings
+
+from .._utils.class_inspect import get_params
+from . import MultiNiftiMasker, NiftiMasker
```

Comment:
Not sure this is good practice to rely on `__init__.py` for these imports. I would rather import from their real home, i.e. `.multi_nifti_masker` and `.nifti_masker`.